frontend/src/scenes/sessionRecordings/sessionRecordingsTableLogic.ts

```diff
@@ -0,0 +1,95 @@
+import { kea } from 'kea'
+import api from 'lib/api'
+import { toParams } from 'lib/utils'
+import { SessionRecordingType } from '~/types'
+import { sessionRecordingsTableLogicType } from './sessionRecordingsTableLogicType'
+import { router } from 'kea-router'
+
+type SessionRecordingId = string
+interface Params {
+    properties?: any
+    sessionRecordingId?: SessionRecordingId
+}
+
+export const sessionRecordingsTableLogic = kea<sessionRecordingsTableLogicType<SessionRecordingId>>({
+    key: (props) => props.personIds || 'global',
```

Comment:
A key should normally return a string, but this might be an array. We're then left to whatever random serialization the JS engine in question uses. Worse case it may be `[Array]` or something simple and unique for all inputs.